.gitignore

```diff
@@ -7,3 +7,4 @@ npm-debug.log
 /.idea
 /node_modules
 /build
+src/data/guides/template.json
```

Comment:
go ahead and check this in if its a useful base for new guides! its okay that its not used at build time

Reply:
I checked it in along with the comma issue from the Ultron guide fixed that caused the build to fail -- the build failed again though and there doesn't appear to be any error message it just disconnected? GitHub had been having some issues when I submitted the update though so maybe just needs to run again?

Reply:
it took longer to run, so it hit some timeout, i increased the karma timeout and its okay